modules/custom/d_product/config/install/image.style.product_main_image.yml

```diff
@@ -4,11 +4,11 @@ dependencies: {  }
 name: product_main_image
 label: 'Product Main Image'
 effects:
-  c7593b10-a82b-42c4-ae9d-1bb38c048a5d:
-    uuid: c7593b10-a82b-42c4-ae9d-1bb38c048a5d
-    id: image_scale_and_crop
-    weight: 1
+  c350d667-1afa-41cb-a52d-f76bb01d8f61:
```

Comment:
Nie wiem czy to dobrze, że te UUID się zmieniły, co jeśli ktoś ma już te style?

Reply:
No szczerze mówiąc też nie wiem czy to dobrze, ale hook update poprawnie się wykonał... Ten UUID jest dla poszczególnego efektu, chyba nie jest do niczego szczególnie potrzebny?

Reply:
hmm, raczej nie jest potrzebny, pytanie tylko co się stanie jak ktoś wykorzystał ten styl w jakimś customowym miejscu - czy nie wyskoczy mu 500

Reply:
To raczej i tak będzie gdzieś referencja do UUID od image stylu, a nie jego efektu. Efekty na każdy styl są unikalne i w sumie nie są nigdzie referowane?